### ISSUE TITLE
Zooming to 200% causes items (e.g. toolbar items) to be offscreen (F69)

### ISSUE THREAD

**Issue by pedr** [laurent22/joplin#11654]
At first, I thought that this might not be addressed because it was already close enough, but it seems like we can only zoom in until 180% or 190% before it starts cutting relevant information.

- [x] Toolbar items from the markdown editor should behave similarly to the rich text editor, when the toolbar requires more space a new option is shown and the new options are hidden on this new submenu.
- [x] Note revision screen also doesn't behave well in the default layout, I think the solution would be to have a horizontal scrollbar

**Comment by personalizedrefrigerator** [comment]
Related: https://github.com/laurent22/joplin/pull/11636

**Comment by pedr** [comment]
Note revision probably can be fixed by moving into a dialog like discussed here:
https://github.com/laurent22/joplin/issues/11648
